### ISSUE TITLE
[REQ] sourceforge - and not only - support

### ISSUE THREAD

**Issue by forart** [badgen/badgen.net#647]
Hi there,
since this project is really cool, we do strongly use it into our GH projects' pages (check out [HyMPS](https://github.com/forart/HyMPS#readme), for example).

It would be even better if you can add other known open source "projects hosting" services, such as sourceforge.

Note that [shields.io already supports some sf "queries"](https://shields.io/search?q=sourceforge), but they don't always work and are quite usless (for our needs).

Hope that inspires !

**Comment by MarcoRavich** [comment]
...but howto ?

<img width="2033" height="1061" alt="Image" src="https://github.com/user-attachments/assets/6070d171-f48b-4cfe-be55-f7d8b0a72cca" />

**Comment by amio** [comment]
Oh I was clearing outdated issues. This should be marked as "not planned".
But if sourceforge has an open api without authorization requirements, then we can do it.

**Comment by MarcoRavich** [comment]
I understand, anyway I think it would have been better to mark as "won't fix"...

**Comment by amio** [comment]
Yeah, I was using batch editing for legacy issues, "not planned" seems like the closest fit.

<img width="230" height="191" alt="Image" src="https://github.com/user-attachments/assets/719ae517-ba17-41f8-96cc-f065433c6d0f" />
